recipe/build.sh

```diff
@@ -15,6 +15,9 @@ if [[ "$target_platform" == linux-ppc64le ]]; then
     EXTRA_CMAKE_FLAGS+=" -D PYTHON_NUMPY_INCLUDE_PATH=${SP_DIR}/numpy/core/include"
 fi
 
+# `cairo` was needed to generate high-quality PNGs for structure depiction,
+# see https://www.rdkit.org/docs/Install.html?highlight=cairo#recommended-extras
+# but we disable it intentionally staring from v2023.03.3
```

Comment:
I'm not sure to understand why we are disabling cairo... Can you explain a little bit more the intent here?

Reply:
that is snowflake requirement to remove cairo-> libx11 dependency

Reply:
Thanks. I think this should be added in the comment then.